libraries/classes/Rte/RteList.php

```diff
@@ -293,6 +293,8 @@ public function getRoutineRow(array $routine, $rowclass = '')
             $routine['type'],
             $routine['name']
         );
+        $params= [];
+        $stmt = (object) [];
```

Comment:
```suggestion
        $stmt = new \stdClass();
```
Would it be better ?

Reply:
i think using (object) [] would be better,It's been used in other files of the codebase too.

Reply:
Okay then, if the codebase uses it